demo_plugins/FileSystemOperations/FileSystemOperations/FileSystemOperations.vcxproj

```diff
@@ -71,7 +71,7 @@
       <GenerateDebugInformation>No</GenerateDebugInformation>
       <LinkTimeCodeGeneration>UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
       <AdditionalLibraryDirectories>$(PLUGIN_SDK_DIR)\output\lib\;$(CLEO_SDK_SA_DIR)\;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
-      <AdditionalDependencies>paths.lib;plugin.lib;cleo.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
any idea why those would be needed? looks like removing them does not affect compilation.

Reply:
No clue too.

Reply:
`plugin.lib` is probably unnecessary since we have a `.cpp`that replicates it. Not sure what is `paths.lib`.

Reply:
ok, thanks